src/components/com_tjcertificate/administrator/languages/en-GB/en-GB.com_tjcertificate.ini

```diff
@@ -0,0 +1,214 @@
+COM_TJCERTIFICATE="TJ Certificate"
+COM_TJCERTIFICATE_TITLE_CERTIFICATES="Templates"
+COM_TJCERTIFICATE_VIEW_CERTIFICATE_TEMPLATES="Templates"
+COM_TJCERTIFICATE_VIEW_CERTIFICATES="Issued Certificates"
+COM_TJCERTIFICATE_TITLE_ISSUED_CERTIFICATES="Issued Certificates"
+
+;Global configuration
+COM_TJCERTIFICATE_CONFIGURATION="TJ Certificate Configuration"
+COM_TJCERTIFICATE_GENEARAL_SETINGS="General"
+COM_TJCERTIFICATE_CERTIFICATE_PREFIX_LABEL="Certificate Prefix"
+COM_TJCERTIFICATE_CERTIFICATE_PREFIX_DESC="Unique Certificate Id Prefix"
+COM_TJCERTIFICATE_CERTIFICATE_RANDOM_STRING_LENTGH_LABEL="Certificate random string length"
+COM_TJCERTIFICATE_CERTIFICATE_RANDOM_STRING_LENTGH_DESC="Random string generated while creating unique certificate id"
```

Comment:
Change this to 

Each generated certificate as a code. The format of the code is [Certificate Prefix]-[Random number]-[Certificate ID]. This option sets the length of the random number.